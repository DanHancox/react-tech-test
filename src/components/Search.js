
// import { useState } from 'react';
import '../styles/search.css';

import React from 'react'

const Search = () => {
  return (
    <>
      render to screen Dvf
    </>
  )
}

export default Search
