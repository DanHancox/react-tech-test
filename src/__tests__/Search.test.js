import { render } from "@testing-library/react";
import React from "react";
import Search from "../components/Search";

describe("renders search to dom", () => {
  it("renders Search", () => {
    const { asFragment } = render(<Search />);

    expect(asFragment()).toMatchSnapshot();
  });
});
