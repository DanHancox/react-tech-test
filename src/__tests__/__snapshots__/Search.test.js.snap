// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders search to dom renders Search 1`] = `
<DocumentFragment>
  render to screen Dvf
</DocumentFragment>
`;
